docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - "9009:9009"
```

Comment:
document ports better:
- 4444 our default spdk no tls before any RPC is available (compose setting for sanity check)
- 5555 our default spdk with tls before any RPC is available (compose setting for sanity check)
- 7777 from cmdline arguments when starting bridge, should be replaced by API change https://github.com/opiproject/opi-api/pull/375